src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1261,6 +1261,16 @@ def _configure_security_policy_storage_params(arg_ctx):
                    'for use with key management services like Azure KeyVault. '
                    'If identity is already assigned - do nothing.')
 
+        # Create args that will be used to build up the Managed Instance's Sku object
+        create_args_for_complex_type(
+            c, 'sku', Sku, [
+                'family',
+                'name',
+                'tier',
+            ])
```

Comment:
Can you show the output of `az sql mi create -h` and make sure that sku-related params have same UX as `az sql db create -h`?

Reply:
I've aligned them 

az sql mi create -h
![image](https://user-images.githubusercontent.com/58415436/72451473-4806b580-37bc-11ea-992c-cfcd7c6dc25b.png)

az sql mi update -h
![image](https://user-images.githubusercontent.com/58415436/72451536-5d7bdf80-37bc-11ea-9bac-f28c6610179d.png)

az sql db create -h
![image](https://user-images.githubusercontent.com/58415436/72451597-771d2700-37bc-11ea-8a4c-fa2e83f68489.png)